client/my-sites/sidebar-unified/style.scss

```diff
@@ -558,6 +558,15 @@ $font-size: rem( 14px );
 // Flyout menu (showing from >782px)
 @media screen and ( min-width: 783px ) {
 	.is-nav-unification {
+		.focus-content,
+		.focus-sites,
+		.focus-sidebar {
+			.sidebar,
+			.layout__secondary {
```

Comment:
Since we have the change in client/layout/style.scss, is it possible to remove .layout__secondary from here, since the layout__secondary is already covered?

Reply:
nice catch! fixed